net/chinadns-ng/Makefile

```diff
@@ -5,53 +5,59 @@
   else ifneq ($(filter $(ARM_CPU_FEATURES),vfp vfpv2),)
-    PKG_SOURCE_URL_FILE:=$(PKG_NAME)+wolfssl@arm-linux-musleabi@generic+v6+soft_float@fast+lto
-    PKG_HASH:=d9095d72dba4018c21ab794c81e467011eee8b3582822a3fa45b803bfdf0d4fa
+    PKG_SOURCE:=$(PKG_NAME)+wolfssl@arm-linux-musleabi@generic+v6+soft_float@fast+lto
+    PKG_HASH:=2e81cef3dae7e736fa98d6a4b2d654c5bbc2e13a0f128f22db564f4cac9a5283
   else
-    PKG_SOURCE_URL_FILE:=$(PKG_NAME)+wolfssl@arm-linux-musleabihf@generic+v7a@fast+lto
-    PKG_HASH:=61fff30848f687d93e58bb91029252818d76e0980d4d32fd2bf6d67dfa51cd4f
+    PKG_SOURCE:=$(PKG_NAME)+wolfssl@arm-linux-musleabihf@generic+v7a@fast+lto
+    PKG_HASH:=7d4f53b8982b415f087fafefb2568a317a0c241d0f79d0cf07ce2de089b11256
   endif
 else ifeq ($(ARCH),i386)
   ifneq ($(CONFIG_TARGET_x86_geode)$(CONFIG_TARGET_x86_legacy),)
-    PKG_SOURCE_URL_FILE:=$(PKG_NAME)+wolfssl@i386-linux-musl@i686@fast+lto
-    PKG_HASH:=8fd1f8cc1ee5f24a10a45367d3444a7af0618f01bf5ea1d14c8bfac856062a23
+    PKG_SOURCE:=$(PKG_NAME)+wolfssl@i386-linux-musl@i686@fast+lto
+    PKG_HASH:=7ad9528daceb81e7e4376d80a6956ca947e6d096f5d3fb3b46a97f64b1c06ddc
   else
-    PKG_SOURCE_URL_FILE:=$(PKG_NAME)+wolfssl@i386-linux-musl@pentium4@fast+lto
-    PKG_HASH:=15983518255abc1849b13352763112442388519109625a9fcd92f50e05831b1c
+    PKG_SOURCE:=$(PKG_NAME)+wolfssl@i386-linux-musl@pentium4@fast+lto
+    PKG_HASH:=1575c0f0223bb95693c3d1d690d6076f50ee91e90e089b422343bf3b403c0f8b
   endif
 else ifeq ($(ARCH),mips)
-    PKG_SOURCE_URL_FILE:=$(PKG_NAME)+wolfssl@mips-linux-musl@mips32+soft_float@fast+lto
-    PKG_HASH:=1b804d4c450d10cb6f91ac0656121bf9fd3e499f15ea569770b8d5523d5290c2
+  ifeq ($(CPU_TYPE),mips32)
+    PKG_SOURCE:=$(PKG_NAME)+wolfssl@mips-linux-musl@mips32+soft_float@fast+lto
+    PKG_HASH:=e5784f4a4bded6a9856469a5d4605c15c1f6d02b13aeb1f46a38fa4f78537bbb
+  else
+    PKG_SOURCE:=$(PKG_NAME)+wolfssl@mips-linux-musl@mips32r2+soft_float@fast+lto
+    PKG_HASH:=a8c0644d5c9715dadafa7890cd24a911718a71f58a4328b426cb9a239c1a2cde
+  endif
 else ifeq ($(ARCH),mipsel)
-  ifeq ($(CONFIG_HAS_FPU),)
-    PKG_SOURCE_URL_FILE:=$(PKG_NAME)+wolfssl@mipsel-linux-musl@mips32+soft_float@fast+lto
-    PKG_HASH:=82d07d148e2c20d4247df7baa0421f1c365954c0953e0e0fbe76e1cd78d1f1b2
+  ifeq ($(CPU_TYPE),)
+    PKG_SOURCE:=$(PKG_NAME)+wolfssl@mipsel-linux-musl@mips32+soft_float@fast+lto
+    PKG_HASH:=129110ee8846323eeec7b44e0227b3b7a2a76cb493c30437a5d65be31e1289c9
+  else ifeq ($(CONFIG_HAS_FPU),)
+    PKG_SOURCE:=$(PKG_NAME)+wolfssl@mipsel-linux-musl@mips32r2+soft_float@fast+lto
+    PKG_HASH:=375453bc696e3cd11dda7476c99aaca0933f6f282a909a2220d459e4624ba550
   else
-    PKG_SOURCE_URL_FILE:=$(PKG_NAME)+wolfssl@mipsel-linux-musl@mips32@fast+lto
-    PKG_HASH:=4a0f66796fbde9c276345ca5e5d4995ce28fef9d54c9538cd931fc2e03de78dd
+    PKG_SOURCE:=$(PKG_NAME)+wolfssl@mipsel-linux-musl@mips32r2@fast+lto
+    PKG_HASH:=fadbf0d08e154922431a29a1d5f63b0347e0728886a23f2fed836c0b908f07f8
```

Comment:
the script `net/chinadns-ng/update.sh` allows you to update hashes easily, don't you even notice this file was removed upstream and the hash was never updated?

Reply:
sorry, I just use the update.sh,to update, but, i didn't check the files one by one upstream to confirm whether that file was removed.